### ISSUE TITLE
datashader kills jupyter notebook kernel

### ISSUE THREAD

**Issue by BKJackson** [pyviz/datashader#812]
I'm trying to run the Holoviz examples tutorial notebook 04_Basic_Plotting.ipynb, and the kernel keeps crashing when I try to run the cell using datashader with the command: 
```
small_df.hvplot.scatter(x='longitude', y='latitude', datashade=True)
```

The pop-up error message in the notebook is "The kernel appears to have died. It will restart automatically."

In the terminal from which I started the ```jupyter notebook``` command, I'm also getting a warning message which may or may not be related:
```
[IPKernelApp] ERROR | No such comm target registered: hv-extension-comm
[IPKernelApp] WARNING | No such comm: hv-extension-comm
```

This occurs both in Safari and Chrome browsers on my Mac OSX.  

I've tried updating all of the holoviz, datashader, etc. libraries.  

I'm running datashader 0.8.0 and holoviews 1.12.6.  

Also, merely importing datashader in the top cell of the tutorial notebook 07_Large_Data.ipynb causes the notebook kernel to crash.
```
import holoviews as hv
import dask.dataframe as dd
import datashader as ds, datashader.geo

from holoviews import opts
from holoviews.operation.datashader import datashade, rasterize

hv.extension('bokeh')
```

**Comment by jbednar** [comment]
Can you report your Numba version?  Also, if `import datashader` alone is sufficient to crash Jupyter, can you try it outside of Jupyter?  I.e., open Terminal, then run `python`, then type that line?  If it crashes, please reply with the output.

**Comment by BKJackson** [comment]
Oh, yes. I almost added that. 

In terminal it gave a segmentation fault 11.

**Comment by BKJackson** [comment]
I tried deleting my holoviz-tutorial environment and reinstalling it from the environment.yml file. It seems to work now except for an error " Config option `use_jedi` not recognized by `IPCompleter`", which I found was a previous issue (https://github.com/pyviz/holoviews/issues/3782).  

I upgraded IPython to 7.5.0 with pip instead of conda, which would've more drastically tried to change the environment.  This may be where things went wrong before--there might be some environment incompatibilities with packages installed by conda.
